src/conn/conn_sweep.c

```diff
@@ -322,27 +322,31 @@ err:		WT_PANIC_MSG(session, ret, "handle sweep server error");
  *	Pull out sweep configuration settings
  */
 int
-__wt_sweep_config(WT_SESSION_IMPL *session, const char *cfg[])
+__wt_sweep_config(WT_SESSION_IMPL *session, const char *cfg[], bool *startp)
 {
 	WT_CONFIG_ITEM cval;
 	WT_CONNECTION_IMPL *conn;
 
 	conn = S2C(session);
 
+	/*
+	 * Sweeping file handles isn't useful if configured in-memory; because
+	 * the default is to configure sweep, turn it off if running in-memory.
+	 */
```

Comment:
I don't follow this -- an application can create and drop tables with `in_memory` configured: why wouldn't we want to clean up obsolete handles?  We don't want to close ordinary handles (hence the previous change forcing `sweep_idle_time=0`), but I think we want sweep to be running.